packages/broker/src/ConfigWizard.ts

```diff
@@ -256,28 +265,42 @@ export const createStorageFile = async (config: any, answers: inquirer.Answers):
     return answers.selectDestinationPath
 }
 
+export const generateMiscFromConfig = (config: any) => {
+    const nodeAddress = new Wallet(config.ethereumPrivateKey).address
+    const mnemonic = Protocol.generateMnemonicFromAddress(nodeAddress)
+    const networkExplorerUrl = `https://streamr.network/network-explorer/nodes/${nodeAddress}`
+    return {
+        mnemonic,
+        networkExplorerUrl
+    }
+}
+
 export const startBrokerConfigWizard = async(): Promise<void> => {
     try {
-        const answers = await inquirer.prompt(prompts)
-        const config = getConfigFromAnswers(answers)
-        const nodeAddress = new Wallet(config.ethereumPrivateKey).address
-        const mnemonic = Protocol.generateMnemonicFromAddress(nodeAddress)
-        logger.info('Welcome to the Streamr Network')
-        logger.print(`Your node's generated name is ${mnemonic}.`)
-        logger.print('View your node in the Network Explorer:')
-        logger.print(`https://streamr.network/network-explorer/nodes/${nodeAddress}`)
-        logger.info('This is your node\'s private key. Please store it in a secure location:')
-        logger.alert(config.ethereumPrivateKey)
+        const privateKeyAnswers = await inquirer.prompt(privateKeyPrompts)
+        const privateKey = getPrivateKeyFromAnswers(privateKeyAnswers)
+        if (privateKeyAnswers.revealGeneratedPrivateKey) {
+            logger.info(`This is your node\'s private key: ${privateKey}`)
+        }
+        const pluginPrompts = createPluginPrompts()
```

Comment:
Could remove internal variables which are just aliases to direct function calls, e.g. `pluginPrompts`.